lib/Core/loadPlugins.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  TerriaPlugin,
+  createPluginContext,
+  ViewState
+} from "terriajs-plugin-api";
+
+type PluginModule = { default: TerriaPlugin };
+
+/**
+ * Load plugin modules.
+ *
+ * @param pluginPromises Array of promise returning plugin modules.
+ */
+async function loadPlugins(
+  viewState: ViewState,
```

Comment:
Will we ever want to expose more than the ViewState here? If so, I'd make a PluginContext interface like:
```ts
export interface PluginContext {
  viewState: ViewState
}
```

Reply:
Yep, that's actually what `createPluginContext` does, it returns a `PluginContext` which is defined in the `plugin-api` repo.